webrender/src/render_backend.rs

```diff
@@ -754,6 +757,13 @@ impl RenderBackend {
         // inflight messages, otherwise the scene builder might panic.
         while let Ok(msg) = self.scene_rx.recv() {
             match msg {
+                SceneBuilderResult::FlushComplete(tx) => {
+                    // If somebody's blocked waiting for a flush, how did they
+                    // trigger the RB thread to shut down? This shouldn't happen
+                    // but handle it gracefully anyway.
+                    debug_assert!(false);
+                    let _ = tx.send(());
```

Comment:
Another way to write that you don't care about the result here is ```tx.send(()).ok();```. I'm not sure which is actually more preferred, so feel free to leave as is if you like.

Reply:
Ah, this is cleaner. Will do this, thanks!